data_collection/gazette/spiders/ba/ba_itaquara_2024.py

```diff
@@ -0,0 +1,11 @@
+from datetime import date
+
+from gazette.spiders.base.brtransparencia import BaseBrTransparenciaSpider
+
+
+class BaItaquaraSpider(BaseBrTransparenciaSpider):
+    name = "ba_itaquara_2024"
+    TERRITORY_ID = "2916708"
+    allowed_domains = ["www.itaquara.ba.gov.br", "api.brtransparencia.com.br"]
+    start_urls = ["https://www.itaquara.ba.gov.br/diario.html"]
+    start_date = date(2019, 1, 1)
```

Comment:
@talesmota pesquisando no site deste raspador , o mais antigo encontrado foi de `26/07/2019`
```suggestion
    start_date = date(2019, 7, 26)
```